features/user-select.yml

```diff
@@ -0,0 +1,16 @@
+name: user-select
+description: The `user-select` CSS property controls which elements can be selected by the user.
+spec: https://drafts.csswg.org/css-ui-4/#content-selection
+caniuse: user-select-none
+group: css
+# `contain` does not have any support yet, so compute from `user-select` key.
+# Not baseline due to `-webkit` prefix in Safari.
```

Comment:
I don't care about the exact wording here, but I'm suggesting elaborating a little bit since I initially didn't understand these as two separate items.

```suggestion
# compute_from: required because `css.properties.user-select.contain` has no support (except historically in IE)
# Baseline status may be surprising. It's caused by a prefix in Safari.
```